test/common/verify_po_resource_stability.go

```diff
@@ -0,0 +1,55 @@
+package common
+
+import (
+	goctx "context"
+	"fmt"
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
```

Comment:
You would also need to add an import for metav1 if changing the `getServiceMonitor()` function
```suggestion
	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```